src/AELFFaucet.Application/Project/ClaimService.cs

```diff
@@ -1,38 +1,96 @@
 using System;
 using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
-
+using Microsoft.Extensions.Options;
 namespace AELFFaucet.Project;
 
 public class ClaimService : AELFFaucetAppService, IClaimService
 {
     private readonly ISendTokenInfoRepository _sendTokenInfoRepository;
     private readonly ITokenSendContractService _otherTokenSendContractService;
     private readonly ITokenSendContractService _nftSeedTokenSendContractService;
-
+    private readonly ApiConfigOptions _apiConfig;
     public ClaimService(ISendTokenInfoRepository sendTokenInfoRepository,
                         OtherTokenSendContractService otherTokenSendContractService,
-                        NftSeedTokenSendContractService nftSeedTokenSendContractService)
+                        NftSeedTokenSendContractService nftSeedTokenSendContractService,
+                         IOptions<ApiConfigOptions> apiConfig)
     {
         _sendTokenInfoRepository = sendTokenInfoRepository;
         _otherTokenSendContractService = otherTokenSendContractService;
         _nftSeedTokenSendContractService = nftSeedTokenSendContractService;
+        _apiConfig = apiConfig.Value;
+    }
+
+    // Method to retrieve the reCAPTCHA secret key based on the platform
+    private string GetRecaptchaSecretKey(string platform)
+    {
+        return platform switch
+        {
+            "FaucetUI" => _apiConfig.FaucetUI,
+            "PlaygroundStaging" => _apiConfig.PlaygroundStaging,
+            "PlaygroundProduction" => _apiConfig.PlaygroundProduction,
+            "AelfStudio" => _apiConfig.AelfStudio,
+            _ => null
+        };
+    }
+
+    private async Task<bool> VerifyRecaptchaAsync(string recaptchaToken, string secretKey)
+    {
+        var url = $"https://www.google.com/recaptcha/api/siteverify?secret={secretKey}&response={recaptchaToken}";
```

Comment:
put the link `https://www.google.com/recaptcha/api/siteverify` in appsettings.json